transports/sqs/index.md

```diff
@@ -80,14 +80,40 @@ NOTE: NServiceBus will perform [immediate](/nservicebus/recoverability/#immediat
 
 ### AmazonSQSException: Request is throttled
 
-SQS can throttle some send operations. When that happens the following exception is logged.
+Amazon SQS could apply throttling, it can deal with a very large continious throughput but if there are sudden spikes the service can apply throttling.
+
+When throttling happens the following exception is logged:
+
 ```
 2017-11-14 23:10:24,314|ERROR|18|NServiceBus.Transports.SQS.MessageDispatcher|Exception from Send.
 Amazon.SQS.AmazonSQSException: Request is throttled. ---> Amazon.Runtime.Internal.HttpErrorResponseException: The remote server returned an error: (403) Forbidden. ---> System.Net.WebException: The remote server returned an error: (403) Forbidden.
 ```
 
-For incoming messages this can be safely ignored as [recoverability feature](/nservicebus/recoverability/) should resolve it. For outgoing messages sent outside of a handler context, send operation can fail. Sending messages in bulk increases the chance of being throttled on send.
+Throttling is more likely to happen when sending a large number messages concurrently (in parallel). For example when using async/await, using a list of tasks to be executed with `Task.WaitAll()` could cause throttling and partial message delivery if any of the sends result in an thottling error.
```

Comment:
I'd remove the `(in parallel)` part. 

I'm not sure we need to be so specific and mention `a list of tasks with Task.WaitAll`. Does it really help to add this. I think sending messages concurrently is enough but that is just my 2 cents.

If you leave it as is at least replace `Task.WhenAll`